rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -87,7 +87,106 @@ in the `indices.get_settings` API.
+
+A `skip` section, on the other hand, defines certain conditions that, if met, will skip the test, such as:
+
+- `cluster_features` to <<cluster_features, skip if cluster features are present>>.
+- `known_issues` to <<skip_known_issues, skip on known issues (based on cluster features)>>.
+- `awaits_fix` to <<skip_awaits_fix, always skip / mute a test due to a pending fix>>.
+- `os` to <<skip_os, skip on operating systems>>.
+- `features`: Only kept for a transition period, please use <<requires_test_runner_features, `test_runner_features`>>
+  in the `requires` section instead.
+
+A `skip` section must specify at least one of the options above.
+Unless only legacy test runner `features` are required, a `reason` must also be provided.
+
+`requires` and / or `skip` can also be used at the top level of the file in the `setup` and `teardown` blocks,
+so all the tests in a file will be skipped if either any requirement fails or any skip condition applies regardless
+if defined in `setup` and `teardown`.
+
+[[cluster_features]]
+=== Require or skip cluster features
+
+In the past, Elasticsearch has been over-reliant on release version checks all across its code base.
+This has become a limiting factor when deploying Elasticsearch in a more continuous fashion.
+
+Moving forward, the goal is to base any such check on the availability of a cluster feature.
+Rather than <<skip_version, skipping tests based on various release version ranges>>, you can
+require `cluster_features` to be either present (`requires`) and / or absent (`skip`) in order to run a test.
+For instance:
+
+....
+    "Parent":
+     - requires:
+          cluster_features: feature_x
+          reason:           Feature X was introduced
+     - skip:
+          cluster_features: feature_x_changed
+          reason:           Change to feature X breaks this test
+
+     - do:
+       ... test definitions ...
+....
+
+The `cluster_features` field can either be a string or an array of strings.
+
+[[synthetic_cluster_features]]
+Note: In order to smoothen the transition from version checks to cluster feature checks, a REST-test specific
+synthetic cluster feature named `gte_v{VERSION}` is available for all release versions until including 8.14.0.
+For instance, `gte_v8.12.2` would be available for all release versions greater than or equal to 8.12.2.
+
+[[skip_known_issues]]
+=== Skip on known issues
+
+Previously, it was possible to skip ranges of broken release versions using <<skip_version,`version`>>.
+`known_issues` provides a more explicit way to express and skip a certain range of buggy releases based on cluster features.
+Each of possibly multiple issues is a pair of `cluster_feature` and `fixed_by`, where an issue was
+introduced by the former feature and eventually fixed by the latter one. For instance:
+
+....
+    "Parent":
+     - skip:
+          known_issues:
+            - cluster_feature: feature_y
+              fixed_by:        feature_y_fix
```

Comment:
May be worth making clear these can be version features too

Reply:
I'm mentioning that just below the example 👍